ci-operator/step-registry/cucushift/installer/check/nutanix/zones/cucushift-installer-check-nutanix-zones-commands.sh

```diff
@@ -13,17 +13,35 @@ declare prism_element3
 # shellcheck source=/dev/null
 source "${NUTANIX_AUTH_PATH}"
 
-node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
-
 if [[ "$SINGLE_ZONE" == "true" ]]; then
+    node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
     echo "Check all nodes running on single zone: $prism_element1"
     for zone in ${node_zone_list}; do
         if [ "$zone" != "$prism_element1" ]; then
             echo "Fail: fail to check single zone: $zone, expected: $prism_element1"
             exit 1
         fi
     done
+elif [[ "$COMPUTE_ZONE" != "" ]]; then
+    compute_pes=$(echo "$COMPUTE_ZONE" | sed -e "s/failure-domain-1/Development-STS/g" -e "s/failure-domain-2/Development-zonal/g" -e "s/failure-domain-3/Development-zonal2/g")
```

Comment:
we are not supposed to use hard code `Development-STS`, `Development-zonal`, and `Development-zonal2` here, but use `prism_element1`, `prism_element2`, `prism_element3` here, right?

Reply:
yes, I'll replace the hard code